Userland/Games/Minesweeper/MinesweeperCustomGameWindow.gml

```diff
@@ -0,0 +1,78 @@
+@GUI::Widget {
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [4]
+    }
+
+    @GUI::GroupBox {
+        title: "Field"
+        autosize: true
+
+        layout: @GUI::HorizontalBoxLayout {
+            margins: [16, 6, 6]
+        }
+
+        @GUI::Label {
+            text: "Columns: "
+            autosize: true
+        }
+
+        @GUI::SpinBox {
+            name: "columns_spinbox"
+            min: 10
+            max: 50
+            fixed_width: 40
+        }
+
+        @GUI::VerticalSeparator {
+        }
+        
+        @GUI::Label {
+            text: "Rows: "
+            autosize: true
+        }
+
+        @GUI::SpinBox {
+            name: "rows_spinbox"
+            min: 10
+            max: 50
+            fixed_width: 40
+        }
+
+        @GUI::VerticalSeparator {
+        }
+
+        @GUI::Label {
+            text: "Mines: "
+            autosize: true
+        }
+
+        @GUI::SpinBox {
+            name: "mines_spinbox"
+            min: 1
+            max: 2500
+            fixed_width: 50
+        }
+    }
+
+    @GUI::Widget {
+        max_height: 24
+
+        layout: @GUI::HorizontalBoxLayout {
+            margins: [4]
+        }
+
+        @GUI::Button {
```

Comment:
We usually align OK and Cancel buttons to the right, which can be achieved using an empty widget as an expanding spacer:

```suggestion
        @GUI::Widget

        @GUI::Button {
```